spokestack/tts/clients/spokestack.py

```diff
@@ -72,6 +72,38 @@ def synthesize(
 
         return response.iter_content(chunk_size=None)
 
+    def synthesize_url(
```

Comment:
we can remove all this duplication by having `synthesize` call `synthesize_url` and use the result to download the audio

Reply:
yep should have been able to see that one. thanks

Reply:
a lot of times it takes a second pair of eyes